### PR TITLE
V4.1

### PR DESCRIPTION
**Pull request by henkkelder**
I've got the example working. Seems fine on PC. Gives CORS errors op my Android device.

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (ihadeed) [inline .vs/VSWorkspaceState.json]: Could you please remove the .vs directory and/or add it to .gitignore

Comment (henkkelder) [reply .vs/VSWorkspaceState.json]: I did. 
I had to make more changes to get it working on my Android & IOS devices.
Please check my commits. I added comment when I though it was useful

**Comment by henkkelder** [comment]
The v4 branch is a sample project. How do we convert this to a usable component? Or do I not understand how it works?

**Comment by ihadeed** [comment]
@henkkelder it's now published under `ionic-image-loader@beta`

**Comment by henkkelder** [comment]
Nice. I will test it today.

**Comment by henkkelder** [comment]
How do I use the ConfigService? This import doesn’t seem to work:

 

import { ImageLoaderConfigService } from "ionic-image-loader";

**Comment by henkkelder** [comment]
Did you see me previous question?

 

From: Henk Kelder <henk.kelder@solcon.nl> 
Sent: vrijdag 14 december 2018 12:03
To: 'zyra/ionic-image-loader' <reply@reply.github.com>
Subject: RE: [zyra/ionic-image-loader] V4.1 (#224)

 

How do I use the ConfigService? This import doesn’t seem to work:

 

import { ImageLoaderConfigService } from "ionic-image-loader";

**Comment by ihadeed** [comment]
@henkkelder sorry for the late response.

I just fixed it and published `ionic-image-loader@7.0.0-beta.1`.

Thank you for your help

**Comment by henkkelder** [comment]
Modified my code to use the corrected beta. 

Seems to work fine.

 

Groeten,

Henk Kelder

**Comment by henkkelder** [comment]
Happy newyear Ibby,

 

Could you please have a look at my PR for V4.

 

Regards,

Henk Kelder from the Netherlands.

**Pull request merged**